universal/ubi9/Dockerfile

```diff
@@ -0,0 +1,430 @@
+# syntax=docker/dockerfile:1.3-labs
+
+# updateBaseImages.sh can't operate on SHA-based tags as they're not date-based or semver-sequential, and therefore cannot be ordered
+FROM quay.io/devfile/base-developer-image:ubi9-latest
+LABEL maintainer="Red Hat, Inc."
+
+LABEL com.redhat.component="devfile-universal-container"
+LABEL name="devfile/universal-developer-image"
```

Comment:
Should we maybe differentiate the labels between the UBI9-based UDI and the UB8-based UDI? E.g. devfile/universal-developer-image-ubi9? Though I suppose this is already accomplished using different image tags?

Reply:
On second though, I don't think this is necessary since UBI 8 will be deprecated as mentioned [here](https://github.com/devfile/developer-images/pull/190#discussion_r1848855421)